### ISSUE TITLE
'_sigmoid()' and '_tanh()' in pytorch_to_caffe.py

### ISSUE THREAD

**Issue by AppleAndBanana** [xxradon/PytorchToCaffe#64]
I found that the function '_sigmoid() ' and '_tanh()' in pytorch_to_caffe.py has no return. This will result in failure when my net has sigmoid or tanh module. Maybe is this a bug?

**Comment by luameows** [comment]
Trying to use nn.functional.sigmoid instead of nn.Sigmoid can solve this problem.
However, the `_tanh()` may still have some bugs, cause it will generate two output blobs like this.
![image](https://user-images.githubusercontent.com/30432086/88650111-0192fb80-d0fb-11ea-9c04-9ff4f7c9e970.png)

**Comment by Single430** [comment]
Mark，调试了半天，终于找到问题解决了！感谢感谢！！
